integration-cli/docker_cli_events_test.go

```diff
@@ -146,9 +146,21 @@ func TestEventsImageUntagDelete(t *testing.T) {
 func TestEventsImagePull(t *testing.T) {
 	since := time.Now().Unix()
 
-	defer deleteImages("hello-world")
+	defer setupRegistry(t)()
+	repoName := fmt.Sprintf("%v/dockercli/hello-world", privateRegistryURL)
+	defer deleteImages(repoName)
 
-	pullCmd := exec.Command(dockerBinary, "pull", "hello-world")
+	repo := fmt.Sprintf("%v:%v", repoName, "latest")
+	if out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "tag", "busybox", repo)); err != nil {
+		t.Fatalf("Failed to tag image %v: error %v, output %q", repo, err, out)
+	}
+	defer deleteImages(repo)
```

Comment:
doesn't `deleteImages(repoName)` also cover `deleteImages(repo)`?
shouldn't the `"repo:tag"` be using variable name `image` instead of `repo`?